data/json/items/chemicals_and_resources.json

```diff
@@ -558,10 +557,9 @@
     "volume": "250 ml",
     "weight": "460 g",
     "bashing": 1,
-    "ammo_type": "components",
     "phase": "liquid",
     "container": "bottle_glass",
-    "//freezing_point": 25,
+    "freezing_point": 25,
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Hydrochloric_acid#/media/File:Phase_diagram_HCl_H2O_s_l.PNG), hydrochloric acid has sub-zero freezing point at concentration below 70%.

> In the United States, solutions of between 20% and 32% are sold as muriatic acid

That gives a freezing point of -42 to -75 celsius.